### PR TITLE
feat(differences): Put finalized EIP-1559 values

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: DEVRL-958

### CODE REVIEW

**Comment by K-Ho** [comment]
we might tweak these, let's hold off for now.

**Comment by qbzzt** [comment]
> we might tweak these, let's hold off for now.

This is a draft until you tell me otherwise.

**Comment by K-Ho** [comment]
values are in here https://www.notion.so/Bedrock-Config-Values-856031d3d46b463485044b40260dad86 - its 5m target

File: src/docs/developers/bedrock/differences.md [hunk 1]
```diff
@@ -115,7 +115,16 @@ As part of the Bedrock upgrade, we have upgraded the network to support [EIP-155
 There are some differences between Ethereum and Optimism in this regard:
 
 - ETH is not burned. Burning ETH on L2 would only lock it in the bridge forever.
-- The EIP 1559 parameters have different values. Once those values are finalized they will be posted here.
+- Some EIP 1559 parameters are different:
+
+  | Parameter | Optimism value | Ethereum value (for reference) |
+  | - | -: | -: |
+  | Block gas limit | 30,000,000 gas | 30,000,000 gas
+  | Block gas target | 6,000,000 gas | 15,000,000 gas
```
Comment (trianglesphere) [inline src/docs/developers/bedrock/differences.md:123]:
```suggestion
  | Block gas target | 5,000,000 gas | 15,000,000 gas
```

File: src/docs/developers/bedrock/differences.md [hunk 2]
```diff
@@ -115,7 +115,16 @@ As part of the Bedrock upgrade, we have upgraded the network to support [EIP-155
...
+  | EIP-1559 elasticity multiplier | 6 | 2
+  | EIP-1559 denominator | 50 | 8
+  | Maximum base fee increase (per block) | 8% | 12.5%
```
Comment (trianglesphere) [inline src/docs/developers/bedrock/differences.md:126]:
```suggestion
  | Maximum base fee increase (per block) | 10% | 12.5%
```

**Review by trianglesphere — COMMENTED** [review]
not strictly necessary & I'm not sure how to word it, but maybe a callout that there are 6 L2 blocks per L1 block.

File: src/docs/developers/bedrock/differences.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/developers/bedrock/differences.md]: Another typo, fixed it. Oops.

**Review by trianglesphere — APPROVED** [review]

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**